doc/nrf/includes/nrf_cloud_rest_sample_requirements.txt

```
@@ -0,0 +1,38 @@
+.. requirement_keysign_start
+
+This sample uses the `nRF Cloud REST API`_, which requires that your device has a valid signing key installed and registered with `nRF Cloud`_.
+
+.. requirement_keysign_moreinfo_start
+
+(See `nRF Connect for Cloud REST API`_ and `Securely Generating Credentials on the nRF9160`_ for more details on this requirement).
+
+Modem version v1.3.x or later is also required.
+
+* Your device may ship with modem firmware older than v1.3.x, so verify that you have the latest installed.
+* Instructions for updating your device's modem firmware may be found here:
+
+  * `Updating modem firmware on nRF9160 DK <updating nRF9160 DK modem firmware>`_
+  * `Updating modem firmware on Thingy:91 <updating Thingy:91 modem firmware>`_
```

```suggestion
  * :ref:`Updating modem firmware on nRF9160 DK <nrf9160_gs_updating_fw_modem>`
  * :ref:`Updating modem firmware on Thingy:91 <programming_thingy>`
```
As commented in the links.txt file. Remove the respective links from that.